addons/account/models/account_payment_term.py

```diff
@@ -59,19 +59,27 @@ def _compute_example_preview(self):
                     discount_date = info_by_dates['discount_date']
                     amount = info_by_dates['amount']
                     discount_amount = info_by_dates['discounted_amount'] or 0.0
-                    example_preview += f"""
-                        <div style='margin-left: 20px;'>
-                            <b>{i+1}#</b>
-                            Installment of
-                            <b>{formatLang(self.env, amount, monetary=True, currency_obj=currency)}</b>
-                            on 
-                            <b style='color: #704A66;'>{date}</b>
-                    """
                     if discount_date:
-                        example_preview += f"""
-                         (<b>{formatLang(self.env, discount_amount, monetary=True, currency_obj=currency)}</b> if paid before <b>{format_date(self.env, terms[i].get('discount_date'))}</b>)
-                    """
-                    example_preview += "</div>"
+                        example_preview += _("<div style='margin-left: 20px;'> " + \
+                                             "<b>%(count)s#</b> " + \
+                                             "Installment of <b>%(amount)s</b> on " + \
+                                             "<b style='color: #704A66;'>%(date)s</b> " + \
+                                             "(<b>%(before_amount)s</b> if paid before <b>%(before_date)s</b>)" + \
+                                             "</div>",
+                                             count=i + 1,
+                                             amount=formatLang(self.env, amount, monetary=True, currency_obj=currency),
+                                             date=date,
+                                             before_amount=formatLang(self.env, discount_amount, monetary=True, currency_obj=currency),
+                                             before_date=format_date(self.env, terms[i].get('discount_date')))
+                    else:
+                        example_preview += _("<div style='margin-left: 20px;'> " + \
+                                             "<b>%(count)s#</b> " + \
+                                             "Installment of <b>%(amount)s</b> on " + \
+                                             "<b style='color: #704A66;'>%(date)s</b>" + \
+                                             "</div>",
+                                             count=i + 1,
+                                             amount=formatLang(self.env, amount, monetary=True, currency_obj=currency),
+                                             date=date)
```

Comment:
Same here.

I would only replace `if paid before` by `{_("if paid before")}` in the original code.